tagger/train/train.py

```diff
@@ -72,35 +76,59 @@ def save_test_data(out_dir, X_test, y_test, truth_pt_test, reco_pt_test, class_l
 
     print(f"Test data saved to {out_dir}")
 
-def train_weights(y_train, truth_pt_train, class_labels, pt_flat_weighting=True):
+def train_weights(y_train, reco_pt_train, class_labels, weightingMethod = WEIGHT_METHOD):
     """
     Re-balancing the class weights and then flatten them based on truth pT
     """
+    if weightingMethod not in ["none", "ptref", "onlyclass"]:
+        raise ValueError("Oops!  Given weightingMethod not defined in train_weights(). Use either none, ptref, or onlyclass.")
     num_samples = y_train.shape[0]
     num_classes = y_train.shape[1]
 
     sample_weights = np.ones(num_samples)
 
     # Define pT bins
+    # pt_bins = np.array([
```

Comment:
Why is this set of pt bins kept as a comment?

Reply:
I'll remove it.